baseapp-chats/baseapp_chats/permissions.py

```
@@ -3,45 +3,82 @@
+            )
 
-                blocks_qs = Block.objects.filter(
-                    models.Q(actor_id__in=my_profile_ids, target_id__in=participant_profile_ids)
-                    | models.Q(actor_id__in=participant_profile_ids, target_id__in=my_profile_ids)
-                )
+            if blocks_qs.exists():
+                return False
 
-                if blocks_qs.exists():
-                    return False
+            return True
```

Ruff (0.8.2)

Return the condition `not blocks_qs.exists()` directly

Replace with `return not blocks_qs.exists()`

(SIM103)